products/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+from core.models import Base
+
+class Category(Base):
+    name = models.CharField(max_length = 50)
+
+    class Meta:
+        db_table = 'categories'
+
+class ProductOptions(Base):
```

Comment:
1. class명은 단수로 만들어주세요~
2. 테이블 명을 아래와 같이 직접 만들어주세요~

```suggestion
class ProductOption(Base):
    class Meta:
        db_table = 'product_options'
```

Reply:
수정하였습니다 !